robot/impl/resource_manager.go

```diff
@@ -369,6 +369,30 @@ func (manager *resourceManager) ResourceNames() []resource.Name {
 	return names
 }
 
+// ResourceStatuses returns the names of all resources in the manager.
```

Comment:
[nit] Could you leave a comment here or in the function to indicate that this method does not return remote (literal remote nodes) nor internal resources?

Reply:
does this sound good?

```suggestion
// ResourceStatuses returns the names of all resources in the manager, excluding the following types of resources:
// - Resources that represent entire remote machines.
// - Resources that are considered internal to viam-server that cannot be removed via configuration.
```

Reply:
LGTM!